hles/extraction/src/test/scala/org/broadinstitute/monster/dap/RedcapMsgGenerator.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.monster.dap
+
+import upack._
+
+object RedcapMsgGenerator {
+
+  def redcapifyRecords(records: List[Map[String, String]]): Arr = {
```

Comment:
not a huge fan of this function name, open to suggestions

Reply:
`toRedcapFormat`?